yb-voyager/cmd/exportData.go

```diff
@@ -900,11 +900,6 @@ func exportDataOffline(ctx context.Context, cancel context.CancelFunc, finalTabl
 	if source.DBType == POSTGRESQL {
 		//Make leaf partitions data files entry under the name of root table
 		renameDatafileDescriptor(exportDir)
-		//Similarly for the export snapshot status file
-		err = renameExportSnapshotStatus(exportSnapshotStatusFile)
```

Comment:
It seems like previously this rename was done much later (after snapshot was complete), and now we seem to be doing it before we actually start export-data (i.e. in initializeExportTableMetadata). 

Is the behavior not impacted because of this change? If this is the correct behaviour, how was it working before?

Reply:
Yeah, so I also realized this thing yesterday. while I was testing so this is the correct behavior to rename this File in the starting itself because in status we are depending upon this to club the status of all leaf to root. So, yeah before this change we were not properly showing the status of partitions during the export data is happening (we were showing <root_table>-<NOT_STARTED>).

Reply:
Got it, thanks! 
Pls verify this manually for partitioned tables (in pg specifically) since we don't have tests for this. 
cc: @shubham-yb

Reply:
@priyanshi-yb 
I imagine that this logic also kicks in for get-data-migration-report, right? 
Does it make sense to then add a test-case for the display name of partitioned tables in the report? (i.e. assert that it shows `<roottable>(<p1>, <p2>)` and not just `<roottable>` ?

Reply:
> I imagine that this logic also kicks in for get-data-migration-report, right?

Yeah I will verify that as well.


> Does it make sense to then add a test-case for the display name of partitioned tables in the report? (i.e. assert that it shows <roottable>(<p1>, <p2>) and not just <roottable> ?


I think this is in reference to table list above. Yeah, I think we can add. Let me see.

Reply:
Verified `get data-migration-report` and added export-data-status report case for table-list and partition handling.